### ISSUE TITLE
Invalid UTF-8 detected

### ISSUE THREAD

**Issue by elliotvaucher** [hsjoberg/blixt-wallet#421]
Hello everyone, 

I'm trying to recover off-chain funds stuck on a LND payment channel [see here for documentation](https://github.com/lightningnetwork/lnd/blob/master/docs/recovery.md#off-chain-recovery). I followed the process described [here](https://community.getumbrel.com/t/how-to-restore-your-node-wallet/3602) which led me to your solution. 

I'm on an IOS Iphone SE, software version 15.0.2. 

I have the seed and the channel.backup file. 

However, when trying to restore a wallet, I get two errors alternatively : 

- Either it says "Invalid UTF-8 detected" (I tried several times, and I know the seed works, as I was able to restore a wallet, without LND funds, on Bluewallet). 
- Either "ENOENT:no such file or directory, open 'private/var/mobile/Containers/Data/Application/.../channels.backup". 

Anyone has an idea on what's happening here ? 

Thank you for your help.

**Comment by hsjoberg** [comment]
Thank you for opening this issue @elliotvaucher.

There are currently some compatibility issues with the channel backup restoration on Blixt iOS.
The reason is that backups imported and exported in Blixt iOS are base64-encoded text strings, whereas this normally are just binary files.
You can either try to convert the file to base64 or restore on an Android phone, which will accept the file fine as it is.

I plan to fix this issue in the next update of Blixt.

Regarding `ENOENT:no such file or directory`, I'm unsure the cause of this. Would be nice to be able to reproduce it to look into it further.

**Comment by elliotvaucher** [comment]
@hsjoberg Thanks for your quick and precise answer. Really appreciate. 

I tried with an Android device. Seems to work. I'm now waiting for the syncing to take place. I will keep you up. 

For the other error, I can provide the following steps : 

- IOS Iphone SE, software version 15.0.2
- Blixt wallet via Apple Testflight
- Target file was saved on the memory of my phone (I previously thought it could be an issue related to accessing a file via iCloud Drive, where it was stored, but then I moved the file on my phone and same error)
- Opening Blixt
- Choosing "restore wallet"
- Entering seed
- Adding file
- Clicking "Restore wallet"

Again, either I had the "utf-8" error, or the "ENOENT", almost alternatively but it was not systematic. 

Thanks again. Have a nice day !
